src/cython/sktda/vector_methods.py

```diff
@@ -286,7 +292,78 @@ def transform(self, X):
                     bc[k] += 1
 
             Xfit.append(np.reshape(bc,[1,-1]))
-            Xfit = np.concatenate(Xfit, 0)
+
+        Xfit = np.concatenate(Xfit, 0)
+
+        return Xfit
+
+class Entropy(BaseEstimator, TransformerMixin):
```

Comment:
Did you see PR #62 and #72 and the discussions starting at https://lists.gforge.inria.fr/pipermail/gudhi-users/2019-March/000221.html and https://lists.gforge.inria.fr/pipermail/gudhi-users/2019-May/000263.html ? Maybe what you have is what we actually want, but it would be nice to acknowledge them and discuss the advantages.

Reply:
Sorry, I did not know some people were already integrating it in the library. I will check their code